cmd/mqtt2prometheus.go

```diff
@@ -182,3 +189,27 @@ func setupExtractor(cfg config.Config) (metrics.Extractor, error) {
 	}
 	return nil, fmt.Errorf("no extractor configured")
 }
+
+func newTlsConfig(cfg config.Config) *tls.Config {
+	certpool := x509.NewCertPool()
+	pemCerts, err := ioutil.ReadFile(cfg.MQTT.CACert)
+	if err == nil {
+		certpool.AppendCertsFromPEM(pemCerts)
+	}
+
+	cert, err := tls.LoadX509KeyPair(cfg.MQTT.ClientCert, cfg.MQTT.ClientKey)
+	if err != nil {
+		return &tls.Config{}
```

Comment:
Could you please return an error here? A panic is very user unfriendly.

Reply:
I've modified to return errors. thanks.